src/components/picker/types.ts

```diff
@@ -5,6 +5,17 @@ import {ModalTopBarProps} from '../modal/TopBar';
 import {TextFieldMethods, TextFieldProps} from '../textField';
 import {TouchableOpacityProps} from '../touchableOpacity';
 
+export interface CustomTopElementProps {
+  /**
+   * The current multi draft value
+   */
+  value: PickerMultiValue;
+  /*
+   * Set the new multi draft value
+   */
+  setValue: (value: PickerMultiValue) => void;
```

Comment:
Let's remove the `setValue` and only expose the `value`, the user can set their internal state thus affecting the component.